scripts/azureml-assets/azureml/assets/release/test_results_analysis.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+"""Python scripts for comparing test results with next stage config."""
+from pathlib import Path
+import yaml
+from azureml.assets.util import logger
+import argparse
+import azureml.assets as assets
+import azureml.assets.util as util
+
+
+def test_results_analysis(config_file: Path, results_file: Path, asset_dir: Path):
```

Comment:
i remember we discussed how do we fail the job when it's not 100% pass for staging, so we know whether to proceed to prod or not. How is that achieved?

Reply:
If there is a warning saying not all assets are covered, we know it.